templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -91,7 +83,7 @@ parameters:
     default: "public"
   - name: os_image
     type: string
-    default: "Ubuntu 18.04 Data Science VM"
+    default: "Ubuntu 22.04 LTS Data Science VM"
```

Comment:
Didn't you remove the Data Science VM from the option list in template_schema.json?
Also, I don't think there is an Ubuntu 22 DSVM image yet, the latest is Ubuntu 20 (per the [docs](https://learn.microsoft.com/en-us/azure/machine-learning/data-science-virtual-machine/dsvm-ubuntu-intro?view=azureml-api-2)).